plugins/interactive.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
+spec:
+  version: v1.0.3
+  homepage: https://github.com/similarweb/kubectl-interactive
+  shortDescription: A kubectl plugin that provides an interactive wrapper on top of kubectl.
+  description: |
```

Comment:
Please wrap at ~80 chars so that is prints nicely on wide and narrow terminals.

Reply:
done